Feature/transform iso code

kerriannercrawford opened the pull request.

winklerj commented on lib/actions/transform.js:
```diff
@@ -8,14 +11,30 @@ const { JsonataTransform } = require('@elastic.io/component-commons-library');
+    compiledExpression.registerFunction('phoneNumtoIso3', (phoneCode) => lookup.byIso(phoneCode));
+    result = compiledExpression.evaluate(msg);
+    self.logger.info(JSON.stringify(result));
+    self.logger.info(JSON.stringify(lookup.byIso(826)));
```
Was this just for testing?  It looks like a hardcoded phone prefix.

kerriannercrawford replied:
Yep this was just for testing. Removed!

winklerj commented on lib/actions/transform.js:
```diff
@@ -37,7 +56,9 @@ async function processAction(msg, cfg) {
   }
+
   this.logger.info('1 message will be send to the next step');
+  this.logger.info(JSON.stringify(result));
```
Can you add a comment to this log statement for context in the logs? Same for line 24 above

winklerj commented on component.json:
```diff
@@ -11,6 +11,9 @@
           "required": true,
           "viewClass": "JSONataView"
+        },
+        "manualTransform": {
```
Can we change this to extendedFunctions? Right now the terminology is targeted toward the implementation.

winklerj commented on lib/actions/transform.js:
```diff
@@ -58,7 +57,7 @@ async function processAction(msg, cfg) {
 
   this.logger.info('1 message will be send to the next step');
-  this.logger.info(JSON.stringify(result));
+  this.logger.info('result is: ', JSON.stringify(result));
```
It's helpful for this comment to be different than line 24 so you can distinguish them in the log. Since you are using the logger, the line number will probably be there anyways. So this is really a small thing.

kerriannercrawford replied:
Appreciate the small things!

kerriannercrawford commented:
Made some small changes to variable names after looking at it again this morning in addition to adding to the logs.

winklerj reviewed (changes requested):
Based on @ryanlunka's feedback, please update the README with instructions how to configure and what the available JSONata functions are.

kerriannercrawford commented:
Added instructions to the README. Open to any feedback. Also updated the return value from the functions - realized they were originally returning the full country object, adjusted so it only returns the ISO3.
